### ISSUE TITLE
Investigate signing Audacium installers

### ISSUE THREAD

**Issue by jt15s** [SartoxSoftware/audacium#83]
Would be good if we could investigate signing the Audacium installers for Windows and maybe MacOS (not too sure how MacOS works though, do they have installers similar to Windows?).

I'm aware that certificates are apparently quite expensive, but it's good to see what the options are and how we could possibly get the installers signed so operating systems don't display any safety warnings for the installer.

**Comment by AnErrupTion** [comment]
Yeah, well, that's going to cost money, and for now this is not possible. This is staying open though.
